.travis.yml

```diff
@@ -34,12 +34,29 @@ before_script:
   # Export database variable for kernel tests.
   - export SIMPLETEST_DB=mysql://root:@127.0.0.1/og
 
+  # Composer.
+  - sed -i '1i export PATH="$HOME/.composer/vendor/bin:$PATH"' $HOME/.bashrc
+  - source $HOME/.bashrc
+
+  # Install Composer dependencies.
+  - composer self-update
+
+  # Install code sniffer.
+  - composer global require squizlabs/php_codesniffer
+
+  # Coder.
+  - composer global require drupal/coder:dev-8.x-2.x
+  - ln -s ~/.composer/vendor/drupal/coder/coder_sniffer/Drupal ~/.composer/vendor/squizlabs/php_codesniffer/CodeSniffer/Standards/
+  - phpenv rehash
```

Comment:
It's easier if we simply add a `composer.json` ourselves, in which coder is a dev-dependency. Then we only need to do a `composer install`.